app/controllers/taxons_controller.rb

```diff
@@ -1,119 +1,47 @@
 class TaxonsController < ApplicationController
   helper_method :taxon_overview_and_child_taxons
 
+  layout 'collections'
+
   def show
-    locals = {
-          taxon: taxon,
-          navigation_helpers: navigation_helpers,
+    setup_content_item_and_navigation_helpers(taxon)
+
+        taxon_template =
+          case presented_taxon.rendering_type
```

Comment:
Doesn't `TaxonPresenter::GRID == :grid` ? So then couldn't you just say `taxon_template = presented_taxon.rendering_type || :leaf` ?